attester-chrome.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [ "$TRAVIS" = "true" ]; then
+    exit
+fi
+
+(node node_modules/attester/bin/attester.js test/attester-phantom-complementary.yml --env package.json &) && selenium-java-robot --browser 'chrome'
```

Comment:
Note that you can use the `--robot-browser` parameter of attester instead of invoking yourself the `selenium-java-robot` command.